src/modules/createTodo.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable import/no-cycle */
+import {
+  deleteTodo, toggleCompleteTodo, updateTodo, toDos,
+} from './crud.js';
+import { toDosContainer } from './selectors.js';
+import addDragListeners from './drag.js';
+
+const createTodo = (object) => {
+  // Create
+  const container = document.createElement('div');
+  container.tabIndex = 1;
+  container.classList.add('toDo');
+  container.setAttribute('index', object.index);
+  container.setAttribute('description', object.description);
+  container.setAttribute('completed', object.completed);
+  const left = document.createElement('div');
+  left.classList.add('left');
+  const right = document.createElement('div');
+  right.classList.add('right');
+  const square = document.createElement('i');
+  square.classList.add('fa-2x');
+  const description = document.createElement('input');
+  description.value = object.description;
+  description.classList.add('description');
+  description.tabIndex = -1;
+  if (!object.completed) {
+    square.classList.add('fa-regular');
+    square.classList.add('fa-square');
+    description.classList.remove('finished');
+  } else {
+    description.classList.add('finished');
+    square.classList.add('fa-solid');
+    square.classList.add('fa-check');
+  }
+  const dots = document.createElement('i');
+  dots.classList.add('fa-solid');
+  dots.classList.add('fa-ellipsis-vertical');
+  dots.classList.add('fa-2x');
+  dots.classList.add('dots');
```

Comment:
I think the code can be improved on this node, you can add various classes in just one line using commas in between class names like so:
```sh
 dots.classList.add('fa-solid', 'fa-ellipsis-vertical', 'fa-2x', 'dots');
```
same goes for any `classList` above. 👆